### PR TITLE
Adding openGemini under Database

### PR DESCRIPTION
**Pull request by xiangyu5632**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by xiangyu5632** [comment]
`11:46:39 AM: [ERROR] Deploy logs are currently unavailable. We are working on resolving the issue.`
there only one log, I don't know the reason for the failure.

File: landscape.yml [hunk 1]
```diff
@@ -5044,6 +5044,13 @@ landscape:
             repo_url: https://github.com/YugaByte/yugabyte-db
             logo: yugabytedb.svg
             crunchbase: https://www.crunchbase.com/organization/yugabyte
+          - item:
+            name: openGemini
+            description: openGemini is an open source distributed time series DBMS with high concurrency, high performance, and high scalability
+            homepage_url: http://www.openGemini.org
+            repo_url: https://github.com/openGemini
```
Comment (castrojo) [inline landscape.yml:5051]:
```suggestion
            repo_url: https://github.com/openGemini/openGemini
```

Comment (castrojo) [reply landscape.yml:5051]: I think needs to be a repo not an org because the org is failing the test and the others that pass link to a repo.

**Review by castrojo — DISMISSED** [review]

**Comment by xiangyu5632** [comment]
LGTM

**Pull request merged**